tests/foundry/test/ExpectRevertTest.t.sol

```diff
@@ -120,8 +134,14 @@ contract ExpectRevertTest is Test {
 
     function test_expectRevert_returnValue() public {
         ReverterWithReturn reverter = new ReverterWithReturn();
+
+        vm.expectRevert("Error");
+        bytes memory returnValueBytes = reverter.returnBytesUnless(true);
+        assertEq0(returnValueBytes, "");
+
         vm.expectRevert("Error");
-        bytes memory returnValue = reverter.revertUnless(false);
-        assertEq0(returnValue, "");
+        (uint256 fst, uint256 snd) = reverter.returnTupleUnless(true);
+        assertEq(fst, 0);
+        assertEq(snd, 0);
     }
 }
```

Comment:
How does this connect with the fact that the dummy return data is 512 bytes? Is it that is passes now and would not pass in the previous version?

Reply:
Yes, because previously it would only return 32 bytes, which is not enough for `(uint256, uint256)` (64 bytes)